local-dev/README.md

```diff
@@ -0,0 +1,15 @@
+# Setting up Audit Database
+
+When running locally, the audit endpoints require a Dynamo DB service to be running on port 8000.
+
+The service hosts a database holding a table that stores audit logs.
+
+To make this table available for testing auditing functionality:
+
+1. Start Docker
+2. Start up a local Dynamo DB container:
+```shell
+docker-compose up -d
+```
+3. Create the audit table schema by running [Creation test](test/aws/AuditTrailDBTest.scala#L46).
+4. Populate the table with some test data by running [Insertion test](test/aws/AuditTrailDBTest.scala#L52).
```

Comment:
I think the link we'd want to give people is the following:
https://github.com/guardian/janus-app/blob/main/test/aws/AuditTrailDBTest.scala#L46-L48

This is the test that sets up the database collections, a requirement for running it locally.

Reply:
Fixed the line refs in 5ede52e8e910b401e68a52e640c83e30bc75f10b